app/controllers/collection_versions_controller.rb

```diff
@@ -31,6 +31,9 @@ def edit
     collection_version = CollectionVersion.find(params[:id])
     authorize! collection_version
 
+    # if we end up on the edit page for a first draft (non-deposited collection), we need a separate form
```

Comment:
I'm not sure I understand "we need a separate form". Can you clarify?

Reply:
basically a form with all of the fields instead of being split into details and settings, like we have for a new collection... i can rephrase the comment